endpoints/openrtb2/auction.go

```diff
@@ -1128,16 +1123,27 @@ func setUAImplicitly(httpReq *http.Request, bidReq *openrtb.BidRequest) {
 	}
 }
 
-// Check if a request comes from a Safari browser
-func checkSafari(r *http.Request) (isSafari bool) {
-	isSafari = false
+// parseUserId gets this user's ID  for the host machine, if it exists.
```

Comment:
Nitpicks: Please remove extra space after `user's ID`. Please update comment to use the same casing as the func name (parseUserID  vs parseUserId).

Reply:
Done